index.js

```diff
@@ -1,6 +1,43 @@
+const yaml = require('js-yaml')
 const mergeArrayByName = require('./lib/mergeArrayByName')
 
 module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  robot.on('installation', async context => {
+    const { github, payload } = context
+    const { action, repositories, installation } = payload
+    const { account } = installation
+    const { login: repositoryOwner } = account
+
+    if (action === 'deleted') {
+      robot.log.debug('Integration deleted, returning...')
+      return
+    }
+
+    await Promise.all(repositories.map(async (repository) => {
+      const { name: repositoryName } = repository
+
+      const repo = {
+        owner: repositoryOwner,
+        repo: repositoryName
+      }
+
+      var res
+      try {
+        res = await github.repos.getContents(Object.assign(repo, { path: Settings.FILE_NAME }))
+      } catch (error) {
+        if (error.status !== 404) {
```

Comment:
You should check if the status is `404` otherwise throw the error, so the error can be handled by probot.